modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -226,6 +226,16 @@ public List<ObjectValidationRule> getObjectValidationRules(
 				objectDefinitionId, active));
 	}
 
+	@Override
+	public List<ObjectValidationRule>
+		getObjectValidationRulesByObjectValidationRuleEngine(
+			String engine, long objectDefinitionId) {
```

Comment:
Wouldn't the order here be the same as in the service.xml file?

Reply:
Good catch, thanks @pedro-oliveira446